deployment/cloud-run/secure_api_server.py

```
@@ -22,21 +23,165 @@
+
+def initialize_advanced_models():
+    """Initialize T5 and Whisper models if available (only if not already loaded)"""
+    global t5_summarizer, whisper_transcriber, T5_AVAILABLE, WHISPER_AVAILABLE
+
+    # Initialize T5 model
+    if T5_AVAILABLE and t5_summarizer is None:
+        try:
+            logger.info("Loading T5 summarization model (fallback)...")
+            t5_summarizer = create_t5_summarizer("t5-small")
+            logger.info("✅ T5 summarization model loaded")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)